modules/lvms-installing-logical-volume-manager-operator-using-rhacm.adoc

```diff
@@ -112,73 +116,21 @@ spec:
-      kind: ConfigurationPolicy
-      metadata:
-        name: lvms
-      spec:
-        object-templates:
-           - complianceType: musthave
-             objectDefinition:
-               apiVersion: lvm.topolvm.io/v1alpha1
-               kind: LVMCluster
-               metadata:
-                 name: my-lvmcluster
-                 namespace: openshift-storage
-               spec:
-                 storage:
-                   deviceClasses:
-                   - name: vg1
-                     default: true
-                     deviceSelector: <2>
-                       paths:
-                       - /dev/disk/by-path/pci-0000:87:00.0-nvme-1
-                       - /dev/disk/by-path/pci-0000:88:00.0-nvme-1
-                       optionalPaths:
-                       - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
-                       - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
-                     thinPoolConfig:
-                       name: thin-pool-1
-                       sizePercent: 90
-                       overprovisionRatio: 10
-                     nodeSelector: <3>
-                       nodeSelectorTerms:
-                       - matchExpressions:
-                           - key: app
-                             operator: In
-                             values:
-                             - test1
-        remediationAction: enforce
-        severity: low
 ----
-<1> Replace the key and value in `PlacementRule.spec.clusterSelector` to match the labels set on the clusters on which you want to install {lvms}.
-<2> Optional. To control or restrict the volume group to your preferred devices, you can manually specify the local paths of the devices in the `deviceSelector` section of the `LVMCluster` YAML. The `paths` section refers to devices the `LVMCluster` adds, which means those paths must exist. The `optionalPaths` section refers to devices the `LVMCluster` might add. You must specify at least one of `paths` or `optionalPaths` when specifying the `deviceSelector` section. If you specify `paths`, it is not mandatory to specify `optionalPaths`. If you specify `optionalPaths`, it is not mandatory to specify `paths` but at least one optional path must be present on the node. If you do not specify any paths, it will add all unused devices on the node.
-<3> To add a node filter, which is a subset of the additional worker nodes, specify the required filter in the `nodeSelector` section. {lvms} detects and uses the additional worker nodes when the new nodes show up.
-+
---
-[IMPORTANT]
-====
-This `nodeSelector` node filter matching is not the same as the pod label matching.
-====
---
+<1> Set the `key` field and `values` field in `PlacementRule.spec.clusterSelector` to match the labels that are configured in the clusters on which you want to install {lvms}.
+<2> Namespace configuration.
+<3> `OperatorGroup` CR configuration.
+<4> `Subscription` CR configuration.
```

Comment:
I would suggest placing a "The" before  the specs in <3> and <4>.

Reply:
Updated as per the suggestion.